provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/AddressResponse.java

```diff
@@ -0,0 +1,83 @@
+package gov.va.api.health.providerdirectory.service;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@JsonIgnoreProperties(ignoreUnknown = true)
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
+public final class AddressResponse implements VlerResponse {
+  @JsonProperty("contacts")
+  private List<Contacts> contacts;
+
+  private Error error;
+
+  /** Lazy getter. */
+  public List<Contacts> contacts() {
+    if (contacts == null) {
+      contacts = new ArrayList<>();
+    }
+    return contacts;
+  }
+
+  @Data
+  @Builder
+  @JsonIgnoreProperties(ignoreUnknown = true)
+  @NoArgsConstructor(access = AccessLevel.PRIVATE)
+  @AllArgsConstructor(access = AccessLevel.PRIVATE)
+  @JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
+  public static final class Contacts {
+    /* Map to 'name'. */
+    @JsonProperty("displayname") // Last, First
+    private String displayname;
```

Comment:
This should be `displayName`. We have to match VLER's field names in the `@JsonProperty()` annotations, but we can still name our fields whatever we want.